python/lsst/jointcal/jointcal.py

```diff
@@ -236,9 +252,6 @@ def run(self, dataRefs, profile_jointcal=False):
         associations.CollectLSSTRefStars(refCat, filt)
         associations.SelectFittedStars()
         associations.DeprojectFittedStars()  # required for AstromFit
-        sky2TP = jointcalLib.OneTPPerShoot(associations.TheCcdImageList())
-        spm = jointcalLib.SimplePolyModel(associations.TheCcdImageList(), sky2TP,
-                                          True, 0, self.config.polyOrder)
 
         # TODO: these should be len(blah), but we need this properly wrapped first.
```

Comment:
What I want is `len(associations.refStarList)`, `len(associations.fittedStarList)`, like the `ccdImageList` test there.